From adb72687ccda69b83509f05f80e4a226a2faaf1f Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto
 <GeorgesStavracas@users.noreply.github.com>
Date: Mon, 24 Feb 2020 17:20:19 -0300
Subject: [PATCH] screencast: Bump supported Mutter version to 3 (#280)

With the new Mutter version and the inclusion of pipewire 0.3 support
into Mutter, the reported D-Bus version for the screencast API was
bumped to 3.

Update the supported version here as well. No code changes are necessary
to make it work with the version 3 of the API.
---
 src/gnomescreencast.c | 2 +-
 src/screencast.c      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gnomescreencast.c b/src/gnomescreencast.c
index 3ae5031..0df0f05 100644
--- a/src/gnomescreencast.c
+++ b/src/gnomescreencast.c
@@ -22,7 +22,7 @@
 
 #include <stdint.h>
 
-#define SUPPORTED_MUTTER_SCREEN_CAST_API_VERSION 2
+#define SUPPORTED_MUTTER_SCREEN_CAST_API_VERSION 3
 
 enum
 {
diff --git a/src/screencast.c b/src/screencast.c
index 5418c23..c4c75a0 100644
--- a/src/screencast.c
+++ b/src/screencast.c
@@ -35,7 +35,7 @@
 #include "session.h"
 #include "utils.h"
 
-#define SUPPORTED_MUTTER_SCREEN_CAST_API_VERSION 2
+#define SUPPORTED_MUTTER_SCREEN_CAST_API_VERSION 3
 
 typedef struct _ScreenCastDialogHandle ScreenCastDialogHandle;
 

